foundation-extensions/mobile-core/identity/README.md

```diff
@@ -332,7 +406,7 @@ If your app opens mobile web content, you need to ensure that visitors are not i
 The Mobile SDK generates a unique visitor ID when the app is installed. This ECID is stored in persistent memory on the mobile device and is sent with every hit. The ECID is removed when the user uninstalls the app or when the user sets the Mobile SDK global privacy status to Opt-out.
 
 {% hint style="info" %}
-When the Mobile SDK privacy status is set to Opt-out, and the ECID is removed, a new unique visitor ID \(ECID\) is generated when the user sets the global privacy status to Opt-In.
+When the Mobile SDK privacy status is set to Opt-out, and the ECID is removed, a new unique visitor ID (ECID) is generated when the user sets the global privacy status to Opt-In.
```

Comment:
Small thing - but is it "Opt-out"? Or is it "Opt-Out"?

Since the opt-in is written as "Opt-In", I just want to double check what's correct here.

Reply:
Looking at the documentation for the other extensions, it looks like the privacy status values of `optedin` and `optedout` should be used here instead, in particular because the privacy status values are being referenced.